chain/consensus/mir/rpc/rpc.go

```diff
@@ -0,0 +1,87 @@
+package rpc
```

Comment:
Should we maybe move this `rcp` package to `chain/ipc-agent`? We may want to move it out to its own repo eventually (it is not lotus-specific) and is probably better if we don't keep it as part of mir here.

Reply:
Do we have `chain/ipc-agent`, or do you want me to create that one?

Reply:
We don't have it yet, can you create it?

Reply:
Sure